rfcs/0009-distributed-dns.md

```diff
@@ -0,0 +1,291 @@
+# Distributed DNS Load Balancing
```

Comment:
'0009' [already taken](https://docs.kuadrant.io/architecture/rfcs/0009-defaults-and-overrides/).